src/ruian.ts

```diff
@@ -1,4 +1,5 @@
 import axios from "axios";
+import proj4 from "proj4";
 
 export type WGS84 = { _type: "WGS84" };
```

Comment:
Tohle jsem tady měl nachystané pro případ, že budem nějak míchat WGS84 a S-JTSK, ale když teď tu konverzi vlastně děláme hned při dotazování serveru, tak bysme ty moje šaškárny s generiky mohli vyrazit a nechat jen něco jako `export type WGS84Location = [number, number]`?

Reply:
mě se to celkem líbilo pro případ, že by to někdy do budoucna šlo parametrizovat a měnit tak souřadnicový systém nebo tak něco ...

Reply:
To byl přesně můj záměr, ale držel bych se spíš principu [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). V dohledné době nevidím důvod tam jiné souřadnice dávat a kdybysme je časem chtěli přidat, můžem tohle snadno vylovit z historie a obnovit. Do té doby akorát každý čtenář toho zdrojáku bude bádat nad tím, proč je ten typ tak složitý. Co myslíš?

Reply:
Jo bude to lepší.

To jen programátor ve mě se ozval že generic věci jsou přeci tak estetické ...

Reply:
Znám ten pocit! :) A tohle byly ještě navíc [fantomové typy](https://dev.to/busypeoples/notes-on-typescript-phantom-types-kg9) :( Nevadí, nacpem je někam jinam.

Reply:
Opraveno viz [db4badb](https://github.com/cesko-digital/obce/pull/21/commits/db4badbbd97629d84bfbb91031e3b883f5250a66)